### PR TITLE
Update MetaTags, add Hackathon ogimage

### PR DESCRIPTION
**Pull request by tellaho**
**Adjusts how `<MetaTags />` works slightly.**
- Slightly more composable now; also added a unique Hackathon ogimage. Need to confirm it works (but this PR needs to be merged for it to be hosted on our server).
- Will use [Twitter Card Validator](https://cards-dev.twitter.com/validator) that the og embed looks as expected after this is merged with main.

**Whacked a bunch of unused pages**

---

**Note:** Couldn't figure out how to render a `<title></title>` within the MetaTags component itself; for some reason it won't recognize the title I set unless it's a direct descendant of the `<Head></Head>` component. Leads to the same  `<Head><MetaTags ... /><title>...</title></Head>` being repeated a lot but none of the solutions [I've found](https://stackoverflow.com/a/66417246) actually work. 🤷‍♂️

### CODE REVIEW

File: next.config.js
Comment (eparubrub) [inline next.config.js:10]: @taylorkmho do we still have a "nominate" button? I thought we grouped this with join?

**Review by eparubrub — APPROVED** [review]
left one minor comment, otherwise LGTM, I like all the cutting of unused pages.

As you said you need to merge to actually confirm the ogimage works, lmk if you need another review after the merge.

File: next.config.js
Comment (tellaho) [reply next.config.js:10]: The redirect is there in case (for any odd reason) someone still had that link and tried to go to it. Also, any stale SEO that points to that URL. In both cases we'll redirect them to the next best place; the join page.

Comment (eparubrub) [reply next.config.js:10]: totally makes sense, good safeguard

Comment (tellaho) [reply next.config.js:10]: I imagine we can remove if next.config.js starts to get unwieldy. But for now no harm no foul afaic.

**Pull request merged**
